### ISSUE TITLE
Accessibility – Voice Over does not function, and no Aria Labels

### ISSUE THREAD

**Issue by CajSWehst** [Phan717/web-design-2-independent-project#8]
Problems:
Your voice-over does not work since there are no aria labels leaving the website not accessible to those with visual impairment. 

<img width="452" alt="Screen Shot 2021-12-10 at 1 32 50 PM" src="https://user-images.githubusercontent.com/77281564/145624207-4a095486-9050-49a4-adc6-ed207cd2a53b.png">

Solution:
Add the aria labels with a detailed but brief description of what is required to make the website more accessible to those with visual impairment.
